src/css/profile/mobile/common/timepicker.less

```diff
@@ -22,6 +22,28 @@
 		}
 	}
 
+	&[data-format="24"] {
+		.ui-time-picker-container {
+			width: 30%;
+
+			&-hour::after {
+				content: ":";
+				height: 164 * @px_base;
+				width: 40% / 3;
```

Comment:
Is the width of the specification document divided by 3?

I'm worried if the value after the decimal point is fine even if the width is not correct.

Reply:
Yes, I think there is no way to calculate the correct size for the width.